packages/pinpin/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@web3-storage/pinpin",
+  "version": "1.0.0",
+  "description": "pin pins on pinata",
+  "license": "(Apache-2.0 AND MIT)",
+  "type": "module",
+  "main": "index.js",
+  "bin": {
+    "pinpin": "./src/bin.js"
+  },
+  "scripts": {
+    "start": "DEBUG_HIDE_DATE=true DEBUG=pinpin ./src/bin.js"
```

Comment:
Do we want debug mode in prod?

Reply:
Currently all the logging is done via debug, and it's flakey so it's useful to see what's going on. There should really be some log levels here, so you can see the job is running (info), vs seeing every request (debug), vs something failed (error), but it's really not critical at the moment.